src/lqcut.jl

```diff
@@ -12,6 +12,9 @@ function lq_drift_time_correction(
      DEP_edgesigma::Float64 = 3.0 , mode::Symbol = :gaussian, lower_exclusion::Float64 = 0.005, upper_exclusion::Float64 = 0.98, drift_cutoff_sigma::Float64 = 2.0,
      prehist_sigma::Float64 = 3.0, e_expression::Union{String,Symbol}="e")
 
+    #get energy units to remve later
+    e_unit = unit(first(e_cal))
```

Comment:
This will fail if `e_cal` is empty (because then, there is not `first`). This could be avoided using `unit(eltype(e_cal))` if the type is defined correctly.

Are we sure that `e_cal` is never empty (or throw an error way earlier in the dataflow)?

Reply:
I copied what the "fit_aoe_corrections" in "aoe_fit_calibration.jl" where it is done similar. I can change it to unit(eltype(e_cal)), maybe then it also can be changed there? But i also think if the array is empty, the function will crash during the fitting.

Reply:
As long as we have some checks that ensure that the array is not empty, this can stay.